packages/agent-toolkit/src/types/tools-configuration.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiClientConfig } from '@mondaydotcomorg/api';
+
+export type ToolsConfiguration = {
```

Comment:
not critical but as you can see we didn't separate types in this repo, I think this can be just a file in the "core" filter named monday-agent-toolkit or something